build-tools/packages/build-cli/src/commands/bundleAnalyses/collect.ts

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import path from "path";
+import { existsSync, copySync, readJSONSync } from "fs-extra";
```

Comment:
Oh I see where this is used now... let's add a task to review all the different readjson/writeasync etc. and unify.

Reply:
Created a work item for reviewing file system methods in all commands: 
[AB#1626](https://dev.azure.com/fluidframework/internal/_workitems/edit/1626)